src/main/kotlin/com/github/devcordde/devcordbot/commands/general/TagCommand.kt

```diff
@@ -310,6 +339,45 @@ class TagCommand : AbstractCommand() {
         }
     }
 
+    private inner class AutoHelpCommand : AbstractSubCommand(this) {
```

Comment:
Please remove that command.

Reply:
I don't like the idea of forcing the user to have it's tags permitted for auto-help. IDK why someone would want to block it's tag but I like to give him the option to do so